lib/expressions.js

```diff
@@ -55,9 +55,14 @@ ExpressionParser.prototype = {
   extractBindingExpression: function (text) {
     var match = text.match(/\{\{(.*)\}\}/) || text.match(/\[\[(.*)\]\]/);
     if (match && match.length === 2) {
-      return match[1];
+      match = match[1];
+      if (match.indexOf("::") > -1) {
+        match = match.slice(0, match.indexOf("::"));
+      }
+    } else {
+      match = undefined;
     }
-    return undefined;
+    return match;
```

Comment:
I'd recommend against reusing match for the return value like this. It's not immediately obvious that this function returns either a string or undefined, because match is also an array.

Add a local result variable?